src/components/base/HiddenContent.vue

```diff
@@ -17,7 +17,6 @@ defineProps<{ show: boolean }>();
 .content-container {
   position: relative;
   height: 100%;
-  display: grid;
   place-content: center;
```

Comment:
why remove the grid?

Reply:
May need to zoom and pair on this one for this explanation, but I'll try in this comment! 

So with grid, it ends up looking like this:
<img width="1505" alt="Screen Shot 2022-05-02 at 10 47 30 AM" src="https://user-images.githubusercontent.com/5270855/166254758-ba58150f-9573-4de0-ae5b-b32b2d7bf19c.png">

Although the `content-container` has full width
<img width="1792" alt="Screen Shot 2022-05-02 at 10 52 58 AM" src="https://user-images.githubusercontent.com/5270855/166255888-6c0dcc1d-561b-401b-bab9-a10ca15248f0.png">

The div inside does not have the full width. See here:
<img width="1792" alt="Screen Shot 2022-05-02 at 10 53 20 AM" src="https://user-images.githubusercontent.com/5270855/166255887-9db22a14-3c75-411a-bcfd-4f2be59e7113.png">

And I wasn't able to get that div to take up the whole container... Once i removed grid though from the `content-container` the layout looked okay

Reply:
makes sense - just also remove the `place-content: center` as that only makes sense with grid

Reply:
Removed !